browser-test/src/applicant/northstar_applicant_application_index.test.ts

```diff
@@ -354,6 +356,25 @@ test.describe('applicant program index page', {tag: ['@northstar']}, () => {
           /* isActive= */ true,
         )
       })
+
+      await test.step('add external program with categories', async () => {
+        await enableFeatureFlag(page, 'external_program_cards_enabled')
```

Comment:
question (non-blocking, but might require a new issue): If I understand this test correctly, you need to enable the external program cards flag to create an external program, but not to view them? I think you'll want to update that so that external programs are filtered out from the home page if the flag is off.

Reply:
This was fixed on #10486. Thanks for the catch!